### ISSUE TITLE
Error importing rsuite/dist/rsuite-no-reset.min.css

### ISSUE THREAD

**Issue by instituto-allamo** [rsuite/rsuite#3835]
### What version of rsuite are you using?

5.64.0

### What version of React are you using?

18.3.1

### What version of TypeScript are you using (if any)?

5.3.3

### What browser are you using?

Chrome

### Describe the Bug

On every compilation, my Next js app with Tailwind can't import the rsuite/dist/rsuite-no-reset.min.css.

![Captura de tela de 2024-06-05 17-49-15](https://github.com/rsuite/rsuite/assets/151032429/5b8eef0a-b398-4545-ae59-96b2d5d7f71b)

My default imports from root layout:

```javascript
import { Toaster } from "@/components/ui/toaster";
import { GetCookieProvider } from "@/utils/store/cookiesStore";
import { PermissionsProvider } from "@/utils/store/permissionContext";

import { CookiesProvider } from "next-client-cookies/server";
import { Inter } from "next/font/google";
import { CustomProvider } from "rsuite";
import { ptBR } from "rsuite/esm/locales";
import "./globals.css";
import "rsuite/dist/rsuite-no-reset.min.css";

```

**Comment by simonguo** [comment]
Hi @instituto-allamo, you mean you can't import `.css.map` file?

**Comment by instituto-allamo** [comment]
> Hi @instituto-allamo, you mean you can't import `.css.map` file?

No, the import is the '.css' file as described, but the Next js console on vs code is getting error 404 importing the .min.css.map

**Comment by simonguo** [comment]
What you want to ask is the 404 error of the `.css.map` file? Or the imported `.css` component style is not effective?

If the component style is not effective, please refer to this example: https://github.com/rsuite/rsuite/tree/main/examples/with-nextjs-app

If you want to solve the problem of the 404 of the `.css.map` file, this should be related to the Nextjs CSS Loader configuration. You can try to move `import "rsuite/dist/rsuite-no-reset.min.css";` to `./globals.css` to try it out.

```css
@import "~rsuite/dist/rsuite-no-reset.css";
```

**Comment by instituto-allamo** [comment]
> What you want to ask is the 404 error of the `.css.map` file? Or the imported `.css` component style is not effective?
> 
> If the component style is not effective, please refer to this example: https://github.com/rsuite/rsuite/tree/main/examples/with-nextjs-app
> 
> If you want to solve the problem of the 404 of the `.css.map` file, this should be related to the Nextjs CSS Loader configuration. You can try to move `import "rsuite/dist/rsuite-no-reset.min.css";` to `./globals.css` to try it out.
> 
> ```css
> @import "~rsuite/dist/rsuite-no-reset.css";
> ```

The style is effective.

About the 404 problem, I tried to move the import to globals.css but the warning remais the same.
